### PR TITLE
docs: Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by mattbretl**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+PostGraphQL version (CLI: `postgraphql --version`; Library: package version):
+
+Steps to reproduce (CLI: complete `postgraphql` command line; Library: relevant code):
+```
+
+```
+
+Current behavior:
+
+Expected behavior:
```
Comment (benjie) [inline ISSUE_TEMPLATE.md]: Great; thanks for this!

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+I'm submitting a ... (check one with "x")
+- [ ] bug report
+- [ ] feature request
```
Comment (benjie) [inline ISSUE_TEMPLATE.md]: Please comment the comment with `<!-- (check one with "x") -->`

Comment (benjie) [inline ISSUE_TEMPLATE.md]: Please comment the comment with `<!-- (check one with "x") -->`

See: https://raw.githubusercontent.com/facebook/react/master/.github/ISSUE_TEMPLATE.md

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+***For bug reports, please include:***
```
Comment (benjie) [inline ISSUE_TEMPLATE.md]: Comment this

Comment (benjie) [reply ISSUE_TEMPLATE.md]: Also, instruct the user to delete everything below here if it's not a bug report

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Minimal SQL file that can be loaded into a clean database:
+``` sql
+
+```
+
+PostGraphQL version (CLI: `postgraphql --version`; Library: package version):
```
Comment (benjie) [inline ISSUE_TEMPLATE.md]: Move this above the "For bug reports" line - I pretty much always want to know what version they're running 👍

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (benjie) [reply ISSUE_TEMPLATE.md]: Probably a few other options we need to add here, one of which is simply "Question" (you'll have to look over the issues to figure out any others)

Comment (mattbretl) [reply ISSUE_TEMPLATE.md]: I read through a few pages of recent issues and checked the labels you've been using..  I think bug/feature/question covers it pretty well; let me know if you want to add any others.

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -1,20 +1,23 @@
-I'm submitting a ... (check one with "x")
+I'm submitting a ... <!-- check one with "x" -->
 - [ ] bug report
 - [ ] feature request
+- [ ] question <!-- alternatively, ask on Gitter (https://gitter.im/postgraphql/postgraphql) -->
```
Comment (benjie) [inline ISSUE_TEMPLATE.md]: Nice addition! 👍

**Pull request merged**